src/main/java/org/spongepowered/common/mixin/core/item/inventory/MixinContainer.java

```diff
@@ -80,6 +81,11 @@
     @Shadow
     public abstract Slot getSlot(int slotId);
 
+    @Shadow
+    public ItemStack slotClick(int slotId, int dragType, ClickType clickTypeIn, EntityPlayer player) {
```

Comment:
If the method is public, you can mark it as abstract and avoid the actual method body.